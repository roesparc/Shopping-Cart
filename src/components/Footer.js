import styles from "../styles/Footer.module.css";

const Footer = () => (
  <footer className={styles.footer}>
    <p>
      By{" "}
      <a className={styles.author} href="https://github.com/roesparc/">
        roesparc
      </a>
    </p>
    <a href="https://github.com/roesparc/">
      <i className={`fa-brands fa-github ${styles.githubLogo}`}></i>
    </a>
  </footer>
);

export default Footer;
