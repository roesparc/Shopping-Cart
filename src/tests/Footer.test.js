import { render } from "@testing-library/react";
import Footer from "../components/Footer";

describe("Footer component", () => {
  test("Renders correct static text and links", () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
