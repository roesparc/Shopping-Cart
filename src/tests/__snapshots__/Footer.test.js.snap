// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Renders correct static text and links 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      By
       
      <a
        class="author"
        href="https://github.com/roesparc/"
      >
        roesparc
      </a>
    </p>
    <a
      href="https://github.com/roesparc/"
    >
      <i
        class="fa-brands fa-github githubLogo"
      />
    </a>
  </footer>
</div>
`;
